src/openzeppelin/utils/constants.cairo

```diff
@@ -6,8 +6,9 @@
 // See: https://github.com/starknet-io/SNIPs/pull/24
 const ISRC5_ID: felt252 = 0x3f918d17e5ee77373b56385708f855659a07f75997f365cf87748628532a055;
 
-// Account
-const IACCOUNT_ID: felt252 = 0x36c738c1c375b993078fe6b517d477e5a3c9b104e40c04662c4bdd3e2f5fa4a;
+// StandardAccount
+const ISTANDARDACCOUNT_ID: felt252 =
+    0x2ceccef7f994940b3962a6c67e0ba4fcd37df7d131417c604f91e03caecc1cd;
```

Comment:
I know SNIP6 is a draft, but we should include a reference like with `ISRC5` above

```suggestion
// StandardAccount
// See: https://github.com/starknet-io/SNIPs/pull/27
const ISTANDARDACCOUNT_ID: felt252 =
    0x2ceccef7f994940b3962a6c67e0ba4fcd37df7d131417c604f91e03caecc1cd;
```